src/test/java/uk/gov/companieshouse/exemptions/delta/ConsumerInvalidTopicTest.java

```diff
@@ -59,21 +57,21 @@ public class ConsumerNonRetryableExceptionTest {
     @MockBean
     private ServiceRouter router;
 
+    @MockBean
+    ChsDeltaDeserialiser chsDeltaDeserialiser;
+
     @Test
-    void testRepublishToInvalidMessageTopicIfNonRetryableExceptionThrown() throws InterruptedException, IOException {
+    void testPublishToInvalidMessageTopicIfInvalidDataDeserialised() throws InterruptedException, IOException, ExecutionException {
```

Comment:
This test case overwrites one that should still be valid? Could we have both?

Reply:
Added back in ConsumerNonRetryableExceptionTest (history lost due to previous test being renamed)